packages/wallet/src/services/types.ts

```diff
@@ -18,10 +18,12 @@ export interface Balance extends Cardano.Value {
 export interface TransactionalObservables<T> {
   total$: BehaviorObservable<T>;
   available$: BehaviorObservable<T>;
+  reserved$: BehaviorObservable<T>;
 }
 
 export interface TransactionalTracker<T> extends TransactionalObservables<T> {
   shutdown(): void;
+  setReserved(utxo: Cardano.Utxo[]): void;
 }
```

Comment:
Changes in `TransactionalTracker` and `TransactionalObservables` also affect `BalanceTracker`, but I don't think we need `reserved$` and `setReserved` there, right? Should I make a new interface for `UtxoTracker` that extends `TransactionalTracker` instead?

Reply:
The concept of reserved UTxOs is relevant, as it reveals the spendable balance

Reply:
I think we should have the `TransactionalObservables.unspendable$` since it's relevant for both balance and utxo.

Since unspendable balance is computed from unspendable utxo, `setUnspendable` should only be a method on UtxoTracker.